charts/seds/psql-sed/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: psql-sed
+description: A Helm chart for PostgreSQL Service Endpoint Definition (SED)
+
+
+type: application
+
+version: 1.0.0
+
+
+appVersion: "1.0.0"
+
+kubeVersion: ">=1.20.0"
+
+annotations:
+  charts.openshift.io/provider: RedHat
+  charts.openshift.io/name: PostgreSQL Service Endpoint Definition (SED)
+  charts.openshift.io/supportURL: https://github.com/dperaza4dustbit/helm-chart
```

Comment:
What should we put for this URL on SBO?

Reply:
So, it would be something like `https://redhat-developer.github.io/service-binding-operator/userguide/service-endpoint-definition/psql.html`

Reply:
This PR should also include adding a section `Service Endpoint Definition` in the User's guide explaining the concept of SED itself.

Reply:
Agree